src/actions/profileActions.js

```diff
@@ -0,0 +1,30 @@
+import AsyncActionType from './AsyncActionType';
```

Comment:
The naming convention we have been following for actions is:
```
src/actions/profile.js
```

Reply:
Happy to change it in the interest of current consistency.  That said, having unique filenames really helps when using "Go to file"-style commands in an IDE... would love to bring this up for discussion later when we're solidifying our patterns.

Reply:
Cool. Does your IDE's autocomplete not take into account directories?

Reply:
Does, I believe, but it's less typing without the "index" :)